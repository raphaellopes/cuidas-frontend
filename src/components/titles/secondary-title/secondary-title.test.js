// vendors
import React from 'react';
import renderer from 'react-test-renderer';

// locals
import SecondaryTitle from './index';

const component = props => (renderer.create(
  <SecondaryTitle {...props} />,
));

describe('<SecondaryTitle />', () => {
  test('Should render the component', () => {
    const tree = component({ children: 'my children' }).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
