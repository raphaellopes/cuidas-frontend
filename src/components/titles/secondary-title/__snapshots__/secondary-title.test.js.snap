// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SecondaryTitle /> Should render the component 1`] = `
<h2
  className="sc-bdVaJa eHfXKq"
>
  my children
</h2>
`;
